sdk/identity/identity/package.json

```diff
@@ -119,45 +90,69 @@
     "@azure/msal-browser": "^3.26.1",
     "events": "^3.0.0",
     "jws": "^4.0.0",
-    "open": "^8.0.0",
+    "open": "^10.1.0",
     "stoppable": "^1.1.0",
     "tslib": "^2.2.0"
   },
   "devDependencies": {
-    "@azure-tools/test-recorder": "^3.0.0",
-    "@azure-tools/test-utils": "^1.0.1",
+    "@azure-tools/test-recorder": "^4.1.0",
+    "@azure-tools/test-utils-vitest": "^1.0.0",
     "@azure/dev-tool": "^1.0.0",
     "@azure/eslint-plugin-azure-sdk": "^3.0.0",
     "@azure/keyvault-keys": "^4.2.0",
-    "@types/chai": "^4.1.6",
     "@types/jsonwebtoken": "^9.0.0",
     "@types/jws": "^3.2.2",
-    "@types/mocha": "^10.0.0",
     "@types/ms": "^0.7.31",
     "@types/node": "^18.0.0",
-    "@types/sinon": "^17.0.0",
     "@types/stoppable": "^1.1.0",
     "@types/uuid": "^8.0.0",
-    "chai": "^4.2.0",
+    "@vitest/browser": "^2.1.4",
+    "@vitest/coverage-istanbul": "^2.1.4",
     "dotenv": "^16.0.0",
     "eslint": "^9.9.0",
     "inherits": "^2.0.3",
     "jsonwebtoken": "^9.0.0",
-    "karma": "^6.2.0",
-    "karma-chrome-launcher": "^3.0.0",
-    "karma-coverage": "^2.0.0",
-    "karma-env-preprocessor": "^0.1.1",
-    "karma-junit-reporter": "^2.0.1",
-    "karma-mocha": "^2.0.1",
-    "karma-mocha-reporter": "^2.2.5",
-    "karma-sourcemap-loader": "^0.3.8",
-    "mocha": "^10.0.0",
     "ms": "^2.1.3",
-    "nyc": "^17.0.0",
-    "puppeteer": "^23.0.2",
-    "sinon": "^17.0.0",
-    "ts-node": "^10.0.0",
+    "playwright": "^1.48.2",
     "typescript": "~5.6.2",
-    "util": "^0.12.1"
+    "util": "^0.12.1",
+    "vitest": "^2.1.4"
+  },
+  "type": "module",
+  "tshy": {
+    "exports": {
+      "./package.json": "./package.json",
+      ".": "./src/index.ts"
+    },
+    "dialects": [
+      "esm",
+      "commonjs"
+    ],
+    "esmDialects": [
+      "browser",
+      "react-native"
+    ],
+    "selfLink": false
+  },
+  "exports": {
+    "./package.json": "./package.json",
+    ".": {
+      "browser": {
+        "types": "./dist/browser/index.d.ts",
+        "default": "./dist/browser/index.js"
+      },
+      "react-native": {
```

Comment:
Until we can verify that this works in `react-native`, we should probably not do this just yet.  We might have to restrict these to browser files for example.

Reply:
fixed